src/definition.ts

```diff
@@ -108,8 +108,9 @@ class API {
     const refUrl = url(absPath);
 
     if (
-      absPath.match(/^\//) ||
-      (absPath.match(/^https?:\/\//) && definitionUrl.hostname === refUrl.hostname)
+      absPath.match(/^\//) || // Unix style filesystem path
+      absPath.match(/^[a-zA-Z]+\:\\/) || // Windows style filesystem path
+      (absPath.match(/^https?:\/\//) && definitionUrl.hostname === refUrl.hostname) // Same domain URLs
```

Comment:
Just a naive question: shouldn't we rely on `path.sep` instead, to explicitly identify/remove directory separators accordingly to the current OS?

Reply:
I don't understand your question sorry. We don't remove directory separators (we delegate the relative path work to the `path` nodejs library) what we do is to detect if it's a filesystem path with the regexp

Reply:
🤦 Sorry!